submissions/NoraAshqar/assignment8.js

```diff
@@ -0,0 +1,54 @@
+//Task 1: Fix the Lost Context
+const editor = {
+  title: 'my first blog',
+  getUpperTitle() {
+    return this.title.toUpperCase();
+  }
+};
+
+// Write code that ensures getTitle() works as expected and logs "MY FIRST BLOG". Do not modify the editor object.
+
+const getTitle =editor.getUpperTitle.bind(editor);
+ console.log(getTitle());
+
+ //Task 2: Arrow or Regular?
+ const formHandler = {
+  value: 'initial',
+  onChange(newValue) {
+    this.value = newValue;
+  }
+};
+
+function simulateInputChange(callback) {
+  callback('updated');
+}
+
+simulateInputChange(formHandler.onChange);
+
+// ❓ Now log formHandler.value Your Tasks:
+console.log(formHandler.value);
+//Predict what formHandler.value will be.
+
+// formHandler.value will remain "initial"
+
+//Fix the bug using either .bind() or an arrow function (your choice).
+simulateInputChange(formHandler.onChange.bind(formHandler));
```

Comment:
Please use arrow function as another solution here and explain why it could work